mw-webapp/src/component/table/Table.tsx

```diff
@@ -1,15 +1,24 @@
 import {useEffect, useState} from "react";
-import {flexRender, getCoreRowModel, useReactTable} from "@tanstack/react-table";
+import InProgress from "../inProgress/InProgress";
+import {useLoading} from "./LoadingContext";
+import {
+  flexRender,
+  getCoreRowModel,
+  useReactTable,
+} from "@tanstack/react-table";
```

Comment:
We use absolute path

You shouldn't use component inside table component/ Your task is create InProgress component/ You can don't change table , columns and other file

Reply:
all modifications to Table deleted